.cirrus.yml

```
@@ -80,10 +94,36 @@ validate_task:
   main_script: *main
 
 
+validate_aarch64_task:
+  alias: "validate_aarch64"
+  depends_on:
+    - "build_aarch64"
+  ec2_instance: *standard_build_ec2_aarch64
+  # From this point forward, all cache's become read-only for this run.
+  cargo_cache: *ro_cargo_cache
+  targets_cache: *ro_targets_cache
+  bin_cache: *ro_bin_cache
```

All three of these need to be updated to use distinct `*_aarch64` cache IDs (aka `fingerprint_key`).
```suggestion
  bin_cache: &ro_bin_cache_aarch64
      <<: *ro_bin_cache
      fingerprint_key: "bin_v1_${DEST_BRANCH}_aarch64"
```